### PR TITLE
Add user set up for readonly on arango

### PR DESCRIPTION
**Pull request by JackWilb**
Depends on #29

I noticed during setup with a couple new users that this was missing. I give it a first pass, but I'm open to re-wording if it needs it

### CODE REVIEW

File: quickstart.rst [hunk 1]
```diff
@@ -45,6 +45,20 @@ Launch ArangoDB
...
+   navigate to http://localhost:8529 and log in with the default username and password,
+   "root" and "letmein", respectively.
+5. Now that you're logged in, navigate to the "_system" database and then to "users" on
+   the left panel in the dashboard.
+6. Click "Add User" setting username to "readonly" and password to "letmein". Click
+   create to make the user.
+7. Now click on the user readonly to set its permissions. After clicking readonly, click
+   "permissions". On the permissions page, on the line that starts with "*", click the radio
+   button that corresponds to access. At this point click on "dashboard" to complete the
+   setup.
```
Comment (waxlamp) [inline quickstart.rst:58]: I wonder if there's a script-based way to set this user up. I filed https://github.com/multinet-app/multinet-api/issues/57 to track that.

File: quickstart.rst [hunk 2]
```diff
@@ -45,6 +45,20 @@ Launch ArangoDB
...
    command not found``, make sure that the docker application is running.
+4. When the container is running, you'll need to make a new user on ArangoDB. To do so,
+   navigate to http://localhost:8529 and log in with the default username and password,
+   "root" and "letmein", respectively.
+5. Now that you're logged in, navigate to the "_system" database and then to "users" on
+   the left panel in the dashboard.
+6. Click "Add User" setting username to "readonly" and password to "letmein". Click
+   create to make the user.
+7. Now click on the user readonly to set its permissions. After clicking readonly, click
+   "permissions". On the permissions page, on the line that starts with "*", click the radio
```
Comment (waxlamp) [inline quickstart.rst:56]: Github seems to think the `*` is the beginning of an italicized phrase (the text following it here in the diff view is italic) but the renderer treats it properly. I wonder if you want to go ahead and escape the `*` here--but this is totally non-urgent.

**Review by waxlamp — APPROVED** [review]

File: quickstart.rst (hunk 1 above)
Comment (JackWilb) [reply quickstart.rst:58]: Good idea! We can update that if/when we tackle that

File: quickstart.rst (hunk 2 above)
Comment (JackWilb) [reply quickstart.rst:56]: Let's see how it renders on readthedocs and if it also sees it as italicized, I'll send another PR to escape it

**Pull request merged**